src/SFA.DAS.EmployerCommitmentsV2.Web/Mappers/Cohort/DetailsViewModelMapper.cs

```diff
@@ -3,28 +3,32 @@
 using System.Threading.Tasks;
 using SFA.DAS.Commitments.Shared.Interfaces;
 using SFA.DAS.CommitmentsV2.Api.Client;
+using SFA.DAS.CommitmentsV2.Types.Dtos;
 using SFA.DAS.EmployerCommitmentsV2.Web.Models.Cohort;
+using SFA.DAS.Encoding;
 
 namespace SFA.DAS.EmployerCommitmentsV2.Web.Mappers.Cohort
 {
     public class DetailsViewModelMapper : IMapper<DetailsRequest, DetailsViewModel>
     {
         private readonly ICommitmentsApiClient _commitmentsApiClient;
+        private readonly IEncodingService _encodingService;
 
-        public DetailsViewModelMapper(ICommitmentsApiClient commitmentsApiClient)
+        public DetailsViewModelMapper(ICommitmentsApiClient commitmentsApiClient, IEncodingService encodingService)
         {
             _commitmentsApiClient = commitmentsApiClient;
+            _encodingService = encodingService;
         }
 
         public async Task<DetailsViewModel> Map(DetailsRequest source)
         {
             var cohortTask = _commitmentsApiClient.GetCohort(source.CohortId);
             var draftApprenticeshipsTask = _commitmentsApiClient.GetDraftApprenticeships(source.CohortId);
 
-            await Task.WhenAll(new List<Task> { cohortTask, draftApprenticeshipsTask });
+            await Task.WhenAll(cohortTask, draftApprenticeshipsTask);
 
-            var cohort = await cohortTask;
-            var draftApprenticeships = (await draftApprenticeshipsTask).DraftApprenticeships;
+            var cohort = cohortTask.Result;
+            var draftApprenticeships = draftApprenticeshipsTask.Result.DraftApprenticeships;
```

Comment:
I read an article which said whilst it was perfectly ok to call .Result in the above circumstance the await style was more appropriate and did exactly the same thing (with no additional cost), as the await will just get the result.

The only other benefit is from a personal POV is I usually look twice when I see a .Result whilst the await looks slight more 'normal'